.dockstore.yml

```diff
@@ -2,7 +2,7 @@ version: 1.2
 workflows:
  - name: TheiaEuk_PE
    subclass: WDL
-   primaryDescriptorPath: /workflows/wf_theiaeuk.wdl
+   primaryDescriptorPath: /workflows/wf_theiaeuk_pe.wdl
```

Comment:
do we want to specify this is for illumina pe?

Reply:
I think so. I do not see anyone doing SE fungal sequencing and and ONT workflow would be very different.